src/classes/AccountModelHealthCheckVMapper.cls

```diff
@@ -0,0 +1,192 @@
+* Default Account, Administrative and Household Account Models are valid and active.
+*/
+public virtual with sharing class AccountModelHealthCheckVMapper {
+    /**********************************************************************************
+    * @description Instance for Singleton Pattern
+    **********************************************************************************/
+    @TestVisible
+    private static AccountModelHealthCheckVMapper instance;
+
+    /**********************************************************************************
+    * @description Empty constructor is not public for Singleton Pattern
+    **********************************************************************************/
+    protected AccountModelHealthCheckVMapper() {}
+
+    /**********************************************************************************
+    * @description Static method to get the current instance for Singleton pattern
+    * @return The instance of AccountModelHealthCheckVMapper.
+    **********************************************************************************/
+    public static AccountModelHealthCheckVMapper getInstance() {
+        if (instance == null) {
+            instance = new AccountModelHealthCheckVMapper();
+        }
+
+        return instance;
+    }
+
+    /**********************************************************************************
+    * @description Retrieve the results for a for an AccountModel health check.
+    * @return A HealthCheckGroupVModel representing a HealtCheckDescription for Account.
+    **********************************************************************************/ 
+    public virtual HealthCheckGroupVModel getHealthCheckGroup() {
+        AccountModelSettingsModel acctModSettingMod = this.locateAccountModelSettingsService().getAccountModelSettingsModel();
+        Map<Id, RecordTypeModel> accountRTModelsById = this.locateRecordTypeService().getModelByIdMapForSObjectType(Account.SObjectType);
+
+        String hlthCheckGroupLabel = Label.stgHealthCheckAccountModel;
+        Integer hlthChkGroupPassedChecks = 0;
+        Integer hlthChkGroupTotalChecks = 0;
+
+        List<String> hlthChkGroupExpandedRowsList = new List<String>();
+        List<HealthCheckItemVModel> hlthChkGroupHlthChkItemsList = new List<HealthCheckItemVModel>();
+
+        HealthCheckGroupVModel defaultHlthChkGroupVMod = this.getDefaultAccountModelHealthCheckItems(acctModSettingMod, accountRTModelsById);        
+        
+        hlthChkGroupPassedChecks += defaultHlthChkGroupVMod.passedChecks;
+        hlthChkGroupTotalChecks += defaultHlthChkGroupVMod.totalChecks;
+
+        String hlthChkGroupStatus = this.updateHealthCheckStatus(hlthChkGroupPassedChecks, hlthChkGroupTotalChecks);
+
+        hlthChkGroupExpandedRowsList.addAll(defaultHlthChkGroupVMod.expandedRowsList);
+        hlthChkGroupHlthChkItemsList.addAll(defaultHlthChkGroupVMod.healthCheckItemList);
```

Comment:
You may want to make a list here and then add a loop through that list in the future to process all health checks equally. Then move the status to the very end.